src/detections/chart/use-get-state.js

```diff
@@ -0,0 +1,8 @@
+import { useDisplayCase } from 'detections/hooks/use-display-case'
+import { useContent } from 'detections/hooks/use-content'
+
+export const useGetState = () => {
```

Comment:
Let's rename this to `usePickContent`.

Reply:
Let's also extract all our hooks to a separate `hooks` directory.

Reply:
done